src/components/ui/pagination/pagination.tsx

```diff
@@ -101,7 +103,8 @@ export const Pagination = ({
       <StyledLink
         onClick={onNextClick}
         $disabled={currentPage === amountOfPages}
-        aria-label="Previous Page"
+        aria-label="Next Page"
+        tabIndex={currentPage === amountOfPages ? -1 : 0}
```

Comment:
same as for the other button, we have to use `disabled` attribute here.

Reply:
There is no `disabled` attribute because the StyledLink is based on a link and only buttons can be disabled. I just added aria-hidden which should have the same effect in combination with the other changes I made to the button.